Graph/main.py

```diff
@@ -0,0 +1,85 @@
+class Graph:
+    def __init__(self) -> None:
+        """
+        Time - O(1)
+        Space - O(1)
+        """
+        self.adjacency_list = {}
+
+    def add_edge(self, node_1, node_2) -> None:
+        """
+        Time - O(1)
+        Space - O(1)
+        """
+        if node_1 not in self.adjacency_list:
+            self.adjacency_list[node_1] = []
+        if node_2 not in self.adjacency_list:
+            self.adjacency_list[node_2] = []
+        self.adjacency_list[node_1].append(node_2)
+        self.adjacency_list[node_2].append(node_1)
+
+    def add_vertex(self, vertex) -> None:
+        """
+        Time -
+        Space -
+        """
+        if vertex not in self.adjacency_list:
+            self.adjacency_list[vertex] = []
+
+    def is_empty(self) -> bool:
+        """
+        Time - O(1)
+        Space - O(1)
+        """
+        return len(self.adjacency_list) == 0
+
+    def size_of_vertices(self) -> None:
+        """
+        Time -
+        Space -
+        """
+
+    def size_of_edges(self) -> None:
+        """
+        Time - O(N)
+        Space - O(1)
+        """
+        length = 0
+        for edge in self.adjacency_list:
+            length += len(self.adjacency_list[edge])
+        print(length // 2)
+
+    def remove_edge(self, node_1, node_2) -> None:
+        """
+        Time - O(1)
+        Space - O(1)
+        """
+        if node_1 and node_2 in self.adjacency_list:
```

Comment:
This if condition is wrong.
```suggestion
        if node_1 in self.adjacency_list and node_2 in self.adjacency_list:
```